packages/@okta/i18n/src/properties/login.properties

```diff
@@ -1322,6 +1322,8 @@ oie.invalid.recovery.token = The recovery token is invalid.
 ## Consent
 oie.consent.scopes.admin.title = would like to access:
 oie.consent.scopes.enduser.title = would like to:
+oie.consent.scopes.granular.title = requests access to
```

Comment:
this is not ideal, since this seems to be the middle of a sentence that concatenates with 2 others
Ideally this would be something like `{0} requests access to {1}`. The main reason is that in other languages the order of those parameters might be different than in English. Is there any chance we can change it?

Reply:
Updated to be like `{0} requests access to`, but I don't think we can add the `{1}` because the following text is not actually part of the header view that's being added, it's the user's login which is automatically shown by the SIW